server.js

```diff
@@ -146,7 +146,7 @@ app.get('/api/info', function(req, res) {
 app.get('/api/auth/:path', function(req, res) {
   auth.checkUrl(req, function(err, output) {
     if (err) {
-      return serverUtils.serverError(err, req, res);
+      return serverUtils.error(err, req, res);
```

Comment:
Because the error messages were not being reported properly.  Afaict `serverUtils.error` is only for errors we are not handling.